### ISSUE TITLE
OpenFL 9.2.1 Started clipping tilemaps when in HiDPI

### ISSUE THREAD

**Issue by ljurow** [openfl/openfl#2632]
@joshtynjala Change on Sep 12, 2022 to Context3D.hx caused this problem

https://github.com/openfl/openfl/commit/22ff362236eac1ce8e2eca309ee62f11e19a666a

1) Create a Hi-DPI project `<window allow-high-dpi="true"/>`
2) Set a monitor to Hi-DPI: i.e. on Windows, right click on the desktop, go to "Display Settings", and select a Scale (in "Scale and Layout") of > 100%)
3) Build a tilemap, and put it inside of a parent that does not start at 0,0 (i.e. x = 100). The tilemap will get clipped

```
var shape:Shape = new Shape();
shape.graphics.beginFill(0xFF0000, 0.5);
shape.graphics.drawCircle(200, 200, 200);
shape.graphics.endFill();
var bmd:BitmapData = new BitmapData(400, 400, false, 0xFF0000FF);
bmd.draw(shape);
var tileset:Tileset = new Tileset(bmd, [new Rectangle(0, 0, 400, 400)]);
testTileMap = new Tilemap(400, 400, tileset, true);
testTileMap.addTile(new Tile(0));
var myDisplay:Sprite = new Sprite();
myDisplay.x = 100; // This is the part that causes the clipping
stage.addChild(myDisplay);
myDisplay.addChild(testTileMap);
```

![tilemap clipping](https://user-images.githubusercontent.com/50641800/233514496-386b1529-461d-4ef0-a07c-c1cb59e11690.jpg)

**Comment by joshtynjala** [comment]
I think you linked to the wrong commit. I suspect that you meant this one instead: 971a4c9e43b5472fd84d73920a2b7c1b3d8d9257

The issue should already be fixed in this commit, which will be included in the next update after 9.2.1: da0463a39863cf9caa86fe2d9716d7579251da5a

**Comment by joshtynjala** [comment]
Closing because it should now be fixed in 9.2.2.
